### ISSUE TITLE
Build without GMP

### ISSUE THREAD

**Issue by uranix** [nakatamaho/mplapack#12]
Hello. I've tried disabling GMP and MPFR by setting `--enable-x=no` in

https://github.com/nakatamaho/mplapack/blob/070348e02d0133a14197d2e81cb96b25e3ea55a5/misc/reconfig.ubuntu20.04.sh#L22-L26

and tried to build MPLAPACK by
```
docker image rm <previous image>
docker build -t mplapack:ubuntu2004 -f Dockerfile_ubuntu20.04 . 2>&1 | tee log.ubuntu2004
```
but I still see GMP being compiled. How can I disable GMP completely?

**Comment by nakatamaho** [comment]
could you please show me the config.log? if --enable-test is still yes, MPLAPACK will build MPFR then GMP.

**Comment by uranix** [comment]
Here's output of `head config.log`
```
This file contains any messages produced by compilers while
running configure, to aid debugging if configure makes a mistake.

It was created by mplapack configure 1.0.1, which was
generated by GNU Autoconf 2.69.  Invocation command line was

  $ ./configure --prefix=/home/docker/MPLAPACK --enable-gmp=yes --enable-mpfr=yes --enable-_Float128=yes --enable-qd=yes --enable-dd=yes --enable-double=yes --enable-_Float64x=yes --enable-test=yes
...
```
File is truncated since I've killed the container too early.

Could it be that docker build uses `mplapack/misc/reconfig.ubuntu20.04.sh` from the remote repository, not from the local one (where I've manually patched it?)

**Comment by nakatamaho** [comment]
Hi,
I need a complete config.log to see what the root cause is.

> Could it be that docker build uses mplapack/misc/reconfig.ubuntu20.04.sh from the remote repository, not from the local one (where I've manually patched it?)

Ok, then you can clone my repository and do some try and error.

**Comment by nakatamaho** [comment]
@uranix Just yesterday, I published mplapack 2.0.0-alpha. You may want to try this version instead of the older version.
